tuberculosis/lib/api/videos.dart

```diff
@@ -1,19 +1,50 @@
 import 'dart:async';
 import 'dart:convert';
+import 'package:Tubuddy/quiz/question.dart';
 import 'package:http/http.dart' as http;
 
+class Quiz {
+  final List<Question> questions;
+
+  Quiz({this.questions});
+
+  factory Quiz.fromJson(Map<String, dynamic> json) {
+    var questions = new List<Question>();
+
+    if (json.containsKey('quizzes') && json['quizzes'] != null) {
+      json['quizzes'].forEach((q) {
+        var answers = List<String>();
+        q['answers'].forEach((d) => answers.add(d as String));
```

Comment:
`d` -> `answer`

Waarom `d`? 🤔

Reply:
Omdat json het type dynamic heeft en je moet die zelf casten naar String.
Als je bijvoorbeeld answers.addAll(q['answers']) doet dan gaat Dart zeuren dat q['answers'] List<dynamic> als type heeft. Om één of andere reden kan je q['answers'] niet casten naar List<String> dus vandaar deze lelijke for each.